DTML.EduBot/Dialogs/UserInformation.cs

```diff
@@ -19,12 +19,50 @@ public partial class ChitChatDialog : QnaLuisDialog<object>
         {
             EntityRecommendation entity;
             string BotResponse = BotPersonality.GetRandomGenericResponse() + Shared.ClientNewLine;
+
             if (result.TryFindEntity(BotEntities.Name, out entity))
-                {
+            {
 
                 ChatContext.Username = entity.Entity;
                 BotResponse = BotPersonality.BotResponseToUserName + " " + ChatContext.Username + "! " + Shared.ClientNewLine;
             }
+            else if (result.Entities.Any(e => e.Type == BotEntities.Hobby))
+            {
```

Comment:
empty bodies?

can you add a TODO comment and if you want to implement this in future, separate from this PR, can you  create a new issue and reference it on this pull request?